src/main/webapp/WEB-INF/tags/instructor/course/courseEditInstructorListPanelBody.tag

```diff
@@ -0,0 +1,98 @@
+<%@ tag description="instructorCourseEdit - Panel Heading of Instructor List" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor/course" prefix="course" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+
+<%@ attribute name="instructorPanel" type="teammates.ui.template.CourseEditInstructorPanel" required="true" %>
+
+<div class="panel-body">
+    <form method="post" action="<%=Const.ActionURIs.INSTRUCTOR_COURSE_INSTRUCTOR_EDIT_SAVE%>"
+            id="formEditInstructor${instructorPanel.index}>" name="formEditInstructors" class="form form-horizontal" >
+        <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>" value="${instructorPanel.instructor.courseId}">
+        <c:if test="${not empty instructorPanel.instructor.googleId}" >
+            <input type="hidden" name="<%=Const.ParamsNames.INSTRUCTOR_ID%>" value="${instructorPanel.instructor.googleId}">
+        </c:if>
+        <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" value="${data.account.googleId}">
+    
+        <div id="instructorTable${instructorPanel.index}">
+            <c:if test="${not empty instructorPanel.instructor.googleId}" >
+                <div class="form-group">
+                    <label class="col-sm-3 control-label">Google ID:</label>
+                    <div class="col-sm-9">
+                        <input class="form-control immutable" type="text" id="<%=Const.ParamsNames.INSTRUCTOR_ID%>${instructorPanel.index}"
+                                value="${instructorPanel.instructor.googleId}" maxlength=<%=FieldValidator.GOOGLE_ID_MAX_LENGTH%> 
+                                tabindex="3" disabled="disabled">
+                    </div>
+                </div>
+            </c:if>
+        
+            <div class="form-group">
+                <label class="col-sm-3 control-label">Name:</label>
+                <div class="col-sm-9">
+                    <input class="form-control" type="text" name="<%=Const.ParamsNames.INSTRUCTOR_NAME%>" 
+                            id="<%=Const.ParamsNames.INSTRUCTOR_NAME%>${instructorPanel.index}" value="${instructorPanel.instructor.name}" 
+                            data-toggle="tooltip" data-placement="top" title="Enter the name of the instructor." 
+                            maxlength=<%=FieldValidator.PERSON_NAME_MAX_LENGTH%> tabindex="4" disabled="disabled">
```

Comment:
You may want to fix such occurrences where the value of certain attributes aren't surrounded in quotations marks in this PR as well.
(`maxlength=<%=FieldValidator.PERSON_NAME_MAX_LENGTH%>`)